client/a8c-for-agencies/components/add-new-site-button/site-selector-and-importer.tsx

```diff
@@ -0,0 +1,122 @@
+		description,
+		buttonProps,
+	}: {
+		icon: JSX.Element;
+		iconClassName?: string;
+		heading: string;
+		description: string;
+		buttonProps?: React.ComponentProps< typeof Button >;
+	} ) => {
+		return (
+			<Button { ...buttonProps } className="site-selector-and-importer__popover-button" borderless>
+				<div className={ clsx( 'site-selector-and-importer__popover-button-icon', iconClassName ) }>
+					<Icon className="sidebar__menu-icon" icon={ icon } size={ ICON_SIZE } />
+				</div>
+				<div className="site-selector-and-importer__popover-button-content">
+					<div className="site-selector-and-importer__popover-button-heading">{ heading }</div>
+					<div className="site-selector-and-importer__popover-button-description">
+						{ description }
+					</div>
+				</div>
+			</Button>
+		);
+	};
+
+	const chevronIcon = isMenuVisible ? 'chevron-up' : 'chevron-down';
+
+	return (
+		<>
+			<Button
+				className="site-selector-and-importer__button"
+				ref={ popoverMenuContext }
+				onClick={ toggleMenu }
+			>
+				{ showMainButtonLabel ? translate( 'Add sites' ) : null }
+				<Gridicon icon={ showMainButtonLabel ? chevronIcon : 'plus' } />
+			</Button>
+			<Popover
+				className="site-selector-and-importer__popover"
+				context={ popoverMenuContext?.current }
+				position="bottom right"
+				isVisible={ isMenuVisible }
+				closeOnEsc
+				onClose={ toggleMenu }
+			>
+				<div className="site-selector-and-importer__popover-content">
+					<div className="site-selector-and-importer__popover-column">
+						<div className="site-selector-and-importer__popover-column-heading">
+							{ translate( 'Import existing sites' ).toUpperCase() }
+						</div>
+						{ menuItem( {
+							icon: <WordPressLogo />,
+							heading: translate( 'Via WordPress.com' ),
+							description: translate( 'Import sites bought on WordPress.com' ),
+						} ) }
+						{ menuItem( {
+							icon: <A4ALogo />,
+							heading: translate( 'Via the Automattic plugin' ),
+							description: translate( 'Connect with the Automattic for Agencies plugin' ),
+						} ) }
+						{ menuItem( {
+							icon: <JetpackLogo />,
+							heading: translate( 'Via Jetpack' ),
+							description: translate( 'Import one or more Jetpack connected sites' ),
+						} ) }
+						{ menuItem( {
+							icon: navigation,
+							iconClassName: 'site-selector-and-importer__popover-button-wp-icon',
+							heading: translate( 'Via URL' ),
+							description: translate( 'Type in the address of your site' ),
+						} ) }
+					</div>
+					<div className="site-selector-and-importer__popover-column">
+						<div className="site-selector-and-importer__popover-column-heading">
+							{ translate( 'Add a new site' ).toUpperCase() }
+						</div>
+						{ menuItem( {
+							icon: <img src={ pressableIcon } alt="" />,
+							heading: translate( 'Pressable' ),
+							description: translate( 'Optimized and hassle-free hosting for business websites' ),
+						} ) }
+						{ menuItem( {
```

Comment:
@yashwin, can we make WordPress.com the first option here? 

<img width="310" alt="Screenshot 2024-07-02 at 5 24 16 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/218541b7-359a-49c8-8964-bc0ee1ad37a7">

pfunGA-1Or-p2#comment-3358

Reply:
Makes sense. I'll address this in my next PR.